### PR TITLE
fix cluster-sync for okd

### PR DESCRIPTION
**Pull request by mhenriks**
**What this PR does / why we need it**:

Creating LocalVolume failing because api version updated

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cluster-up/ssh.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ if [[ $provider_prefix =~ okd.* ]]; then
         exit 1
     fi
     shift
-    ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -q -lcore -p $port core@127.0.0.1 -i ${ssh_key} $@
+    ssh -o UserKnownHostsFile=/dev/null -oStrictHostKeyChecking=no -q -lcore -p $port core@127.0.0.1 -i ${ssh_key} $@
```
Comment (awels) [inline cluster-up/ssh.sh]: I have an unpublished PR that uses the latest kubevirtci, and the ssh.sh in there already adds the StrictHostKeyChecking to it. Maybe just wait for that?

Comment (mhenriks) [reply cluster-up/ssh.sh]: Let's see how this run goes had to fix `ssh.sh`

**Comment by mhenriks** [comment]
@awels okd tests running now.  Interestingly, fewer failures than os.

**Comment by awels** [comment]
The failures are due to the cloning issue in os-3.11.

**Comment by mhenriks** [comment]
> The failures are due to the cloning issue in os-3.11.
I know.  My point was that there are fewer failures on okd and that is kind of interesting.

**Comment by awels** [comment]
Yeah seems that whatever broke OS-3.11, didn't break 4.1

**Comment by awels** [comment]
I think #873 includes the contents of this already. And the cluster-up ssh.sh fixes as well.

**Comment by mhenriks** [comment]
> I think #873 includes the contents of this already. And the cluster-up ssh.sh fixes as well.

It does not include the fix to `ssh.sh`

**Comment by awels** [comment]
https://github.com/kubevirt/containerized-data-importer/blob/master/cluster-up/ssh.sh#L43 has the fix, for the ssh already, not sure when that made it in.

**Comment by mhenriks** [comment]
> https://github.com/kubevirt/containerized-data-importer/blob/master/cluster-up/ssh.sh#L43 has the fix, for the ssh already, not sure when that made it in.

No, that is not the fix I've included here.  This line is broken in CI:  https://github.com/kubevirt/containerized-data-importer/blob/master/cluster-up/ssh.sh#L29

**Pull request merged**